### DISCUSSION TITLE
Expo grid mode: add desktop shortcuts workspace up/down

### DISCUSSION THREAD

**Discussion by ghost** [linuxmint/discussions#642] [Ideas]
There is already the workspace grid option in Expo. Why not two additional shortcuts up/down, given the numerous requests? If unassigned by default, they do not change anything for other users. And yet, all feature request are denied.

Changing up/down now is: ctrl+alt+up, screen changes, up or down, screen changes, escape, screen changes.  

By the way, the animations during the first and last screen transitions include a full-screen blink caused by an abrupt change in brightness.

The optional grid workspace applet results in two independent grid configurations. And currently it does not even work properly with Expo.

**Comment by rcalixte** [comment]
@d01010101 If I understand your request correctly, it's already a part of this [Discussion post](https://github.com/orgs/linuxmint/discussions/37)? If so, there isn't a timeline for these changes to happen or even a guarantee.

**Comment by ghost** [comment]
Thanks rcalixte, I did not see it. The details do not seem to be the same, though. What I was asking would not in any way be disruptive and would not require the workspace grid applet.

Combining the grid Expo and the single-line Workspace switcher is already a feature in Cinnamon. While somewhat counter-intuitive, it has the advantage of larger and clearer workspace buttons, given the panel's limited height. Using it with up/down shortcuts seems fine to me.

**Comment by ghost** [comment]
If anyone is experiencing issues with the optional [Workspace Grid (2D) and Switcher](https://cinnamon-spices.linuxmint.com/applets/view/116) applet stopping responding to shortcuts, it's because it reuses the shortcut definitions for the default one-row Cinnamon workspaces. If you unassign a shortcut for the latter, both will lose it.

However, they can coexist through something called "Hot Corners".

While the 2D switcher doesn't display workspace contents, its buttons are so small in 2D mode that it might be for the better - at least it clearly indicates the active workspace. The 2d switcher may seamlessly coexist, though, with the 1d Cinnamon switcher if one needs buttons with visualisation.
